### PR TITLE
refactor: Add option struct to optimize parameter transfer

### PR DESCRIPTION
**Pull request by Forget-C**
Add option struct to optimize parameter transfer

**Affect user experience**： false

### CODE REVIEW

File: pkg/capacity/options.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package capacity
+
+type Option struct {
```
Comment (isaacnboyd) [inline pkg/capacity/options.go:3]: `
$ golangci-lint run

pkg/capacity/options.go:3:6: exported type `Option` should have comment or be unexported (golint)
type Option struct {
     ^
`
there is some lint here

**Comment by isaacnboyd** [comment]
Very cool! This will have conflict with a few of the open PRs as they are all new feature flags. Will need a good rebase after PRs 109, 110, and 113 are merged.

File: pkg/capacity/options.go (hunk 1 above)
Comment (Forget-C) [reply pkg/capacity/options.go:5]: Comments have been added

File: pkg/capacity/options.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package capacity
+
+// Option is a struct containing the command line options
+// FetchAndPrint depends on
+type Option struct {
```
Comment (robscott) [inline pkg/capacity/options.go:5]: Tiny Nit: I think `Options` would be a bit more natural than `Option` here.
```suggestion
type Options struct {
```

File: pkg/capacity/capacity.go [hunk 3]
```diff
@@ -19,44 +19,43 @@ import (
...
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 	v1beta1 "k8s.io/metrics/pkg/apis/metrics/v1beta1"
+	metrics "k8s.io/metrics/pkg/client/clientset/versioned"
 )
 
 // FetchAndPrint gathers cluster resource data and outputs it
-func FetchAndPrint(showContainers, showPods, showUtil, showPodCount, excludeTainted, availableFormat bool, podLabels, nodeLabels, namespaceLabels, namespace, kubeContext, kubeConfig, output, sortBy string) {
-	clientset, err := kube.NewClientSet(kubeContext, kubeConfig)
+func FetchAndPrint(option Option) {
```
Comment (robscott) [inline pkg/capacity/capacity.go:31]: Similar to other comment, recommend using plural names here.
```suggestion
func FetchAndPrint(opts Options) {
```

**Review by robscott — COMMENTED** [review]
Thanks @Forget-C! This is a great improvement that has been long overdue. Just a tiny nit on naming but otherwise LGTM.

**Comment by Forget-C** [comment]
> Thanks @Forget-C! This is a great improvement that has been long overdue. Just a tiny nit on naming but otherwise LGTM.

Thank you for your advice. I modified the naming and merged with the master branch

**Comment by robscott** [comment]
Thanks @Forget-C!

**Pull request merged**